fuel-relayer/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "fuel-relayer"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+repository = "https://github.com/FuelLabs/fuel-core"
+description = "Fuel Relayer"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+bytes = "1.1"
+env_logger = "0.9"
+ethers-core = { git = "https://github.com/rakita/ethers-rs.git", branch = "master", package = "ethers-core", default-features = false }
+ethers-middleware = { git = "https://github.com/rakita/ethers-rs.git", branch = "master", package = "ethers-middleware", default-features = false }
+ethers-providers = { git = "https://github.com/rakita/ethers-rs.git", branch = "master", package = "ethers-providers", default-features = false, features = [
```

Comment:
To publish this crate to crates.io, we need all dependencies to be releases on crates.io. Could be saved for a future PR if you set `publish = false` so we don't try to publish this.

Reply:
Yea, will add `publish = false`. Ethers does not have that up to data version on crates (last version is from december). Will ping them to update it.

Reply:
line added